<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1405</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LOGIN </string>
  </property>
  <property name="windowIcon">
   <iconset resource="bookholics.qrc">
    <normaloff>:/image/1.png</normaloff>:/image/1.png</iconset>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>431</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/logo.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_message">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>660</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>290</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(85, 255, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>401</width>
      <height>93</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_username">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
       </property>
       <property name="placeholderText">
        <string>Enter valid  username </string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::LogicalMoveStyle</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>****************</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Show Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>136</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Captcha</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
padding:4px;</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Login_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);
padding:4px;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>181</width>
      <height>85</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_enter_above_captcha">
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>true</strikeout>
         <stylestrategy>NoAntialias</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_captcha">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
       </property>
       <property name="placeholderText">
        <string>Enter Above Captcha</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="bookholics.qrc"/>
 </resources>
 <connections/>
</ui>
